### PR TITLE
Ported Slimalyzer to 1.16

### PR DESCRIPTION
**Pull request by justliliandev**

### CODE REVIEW

File: OpenBlocks/src/main/java/openblocks/common/tileentity/HealTileEntity.java [hunk 1]
```diff
@@ -18,6 +18,8 @@ public HealTileEntity() {
     @Override
     public void tick() {
         if (world.isRemote) return;
+        ServerWorld serverWorld = (ServerWorld) world;
```
Comment (boq) [inline OpenBlocks/src/main/java/openblocks/common/tileentity/HealTileEntity.java]: Unused

File: OpenBlocks/src/main/resources/assets/openblocks/lang/en_us.json [hunk 2]
```diff
@@ -12,6 +12,7 @@
     "block.openblocks.elevator.description": "Place one elevator directly three or more blocks above another with no blocks inbetween and you'll be able to either 'jump' to move up, or 'crouch' to move down. Elevators can be dyed by clicking on them with dye, however, you can only teleport between elevators of the same color.",
     "block.openblocks.rotating_elevator.name": "Rotating Elevator",
     "block.openblocks.heal": "Healer",
+    "item.openblocks.slimalyzer": "Slimalyzer",
```
Comment (boq) [inline OpenBlocks/src/main/resources/assets/openblocks/lang/en_us.json:15]: Please also grab [description](https://github.com/OpenMods/OpenBlocks/blob/master/src/main/resources/assets/openblocks/lang/en_us.lang#L325), it's not used yet, but will be for guide

**Review by boq — APPROVED** [review]
Couple nits and now it conflicts, since I finally finished tanks (were sitting in my repo for couple month).

**Comment by justliliandev** [comment]
Should be fixed now

**Pull request merged**